guest-owner-proxy/keysets.json

```diff
@@ -0,0 +1,24 @@
+{
+	"KEYSET-1":
+		{ 
+		  	"allowed_digests":[
+				"66320db73158a35a255d051758e95ed4",
+				"5c3292446d5f85a91923f11f8f58452bd380c7fbd12887dbe15fe1f485cdbaf2"
+			],
+			"allowed_keys":[
+				"KEY-ID-1",
+				"KEY-ID-2"
+			],
+			"allowed-policies":[
```

Comment:
Minor observation.. My gut instinct is that a keyset should be independent from SEV policies and build/api versions (i.e., a keyset could be applied to TDX deployment in the future). Is there a particular reason you did it this way?

Reply:
The idea is just that a keyset will only be released to nodes that meet the security requirements of the guest owner. I guess we could have tenant-wide requirements for the nodes instead. Maybe the best thing would be to make the fields optional for an individual keyset and have per-tenant default values.